types/redux-first-router/index.d.ts

```diff
@@ -80,25 +99,29 @@ export interface NavigationAction {
     routeName?: string;
     actions?: NavigationAction[];
     action?: NavigationAction;
-    params?: object;
-    meta?: object;
+    params?: Params;
+    meta?: Meta;
```

Comment:
I don't currently have the time to look at this thoroughly, but is this different from the Flow types of the package intentionally? Over there, `meta` is an `object`.